README.md

```diff
@@ -36,32 +51,21 @@ $ yarn start-docker
 
 **Install**
 
-Using npm:
-
-```shell
-$ npm install
-```
-
 Using yarn:
 
 ```shell
 $ yarn install
 ```
 
-After installing your dependencies, ensure are using the proper version of node by running NVM:
+After installing your dependencies, ensure are using the proper version of node
```

Comment:
Change `ensure are using the proper version of node`  to `ensure you are using the proper version of node`